ls1intum/Artemis
# [Enhancement] Improve grading page

### Checklist

### Motivation and Context



Better explain the options and charts of the grading page.
Resolves #2572 

### Description


- [x] split chart for test-cases
- [x] add better explanations for the charts (+ translations)
- [x] add (?) explanations for weights, bonus multiplier and bonus points
- [x] add (?) explanations for penalty and max penalty
- [x] add explanations to sca category issue chart
- [x] make the tables sortable

### Steps for Testing


1. Log in to Artemis
2. Navigate to the grading page
3. Check that sorting in the test case and category tables works
4. Check the (?) tooltips.
5. Check the chart descriptions.

Please also give Feedback on the hints and descriptions if you have any.

### Test Coverage




Nothing changed.

### Screenshots



<img width="1029" alt="Bildschirmfoto 2021-01-03 um 01 36 49" src="https://user-images.githubusercontent.com/13920539/103469377-32d7bd80-4d64-11eb-8d98-d042cfd357f7.png">
<img width="1054" alt="Bildschirmfoto 2021-01-03 um 03 58 52" src="https://user-images.githubusercontent.com/13920539/103470845-07f76480-4d78-11eb-8dfa-f1f8fdce37da.png">
